sites/public/.env.template

```diff
@@ -13,7 +13,7 @@ CLOUDINARY_CLOUD_NAME=exygy
 CACHE_REVALIDATE=60
 NEW_RELIC_APP_NAME=Bloom Public
 NEW_RELIC_LICENSE_KEY=
-# this GTM key is for local development only
-GTM_KEY=GTM-KF22FJP
+GA_KEY=G-7N2S5HXLQE
```

Comment:
This new key is also just for dev purposes, yeah?

Reply:
Good point! That is the key for Detroit's prod sites so seems best to remove it from the template if you agree?

Reply:
Yep, we wouldn't want to end up in a place where we're changing prod keys locally / have any prod keys in the template.

Reply:
Makes sense! It's been removed 🥦